src/main/java/org/apache/commons/lang3/SortedListUtils.java

```diff
@@ -0,0 +1,89 @@
+package org.apache.commons.lang3;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.function.Function;
+
+
+/**
+ * Operations on sorted {@link List}.
+ *
+ * @author Zbynek Vyskovsky
+ */
+public class SortedListUtils {
```

Comment:
Collection utilities do not belong in Commons Lang IMO. This would be for Commons Collections unless this functionality is already there.

Reply:
I removed it.  Unsure whether this was the best choice, as this is about the algorithm, not about implementing collections.